src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -253,7 +253,7 @@ void Analyzer::Display() {
   // Function that displays a read-only value.
   auto ShowGain = [](const char* text, double* data) {
     ImGui::SetNextItemWidth(ImGui::GetFontSize() * 4);
```

Comment:
We might need to increase this width a little to accommodate the extra significant figures.